cpp/include/cuml/manifold/spectral_embedding.hpp

```diff
@@ -58,4 +58,11 @@ void transform(raft::resources const& handle,
                raft::device_coo_matrix_view<float, int, int, int> connectivity_graph,
                raft::device_matrix_view<float, int, raft::col_major> embedding);
 
+void transform(raft::resources const& handle,
+               ML::SpectralEmbedding::params config,
+               raft::device_vector_view<int, int> rows,
+               raft::device_vector_view<int, int> cols,
+               raft::device_vector_view<float, int> vals,
+               raft::device_matrix_view<float, int, raft::col_major> embedding);
```

Comment:
It looks like the API won't work with datasets having more elements (nnz) than `std::numeric_limits<int>::max`. Would be great to update the cuVS and cuML APIs to allow larger matrices (extent as `uint64_t`). Maybe as a follow-up PR?